src/overlays/actors/ovl_Bg_Mizu_Movebg/z_bg_mizu_movebg.c

```diff
@@ -82,84 +92,85 @@ void BgMizuMovebg_Init(Actor* thisx, PlayState* play) {
     s32 type;
     s32 waypointId;
     WaterBox* waterBoxes = play->colCtx.colHeader->waterBoxes;
-    f32 temp;
+    f32 targetPosY;
     CollisionHeader* colHeader = NULL;
-    Vec3f sp48;
+    Vec3f offsetPos;
 
     Actor_ProcessInitChain(thisx, sInitChain);
     ((BgMizuMovebg*)thisx)->homeY = thisx->world.pos.y;
-    ((BgMizuMovebg*)thisx)->dlist = D_8089EB50[MOVEBG_TYPE(thisx->params)];
+    ((BgMizuMovebg*)thisx)->dlist = sDLists[MOVEBG_TYPE(thisx->params)];
     DynaPolyActor_Init(&((BgMizuMovebg*)thisx)->dyna, DYNA_TRANSFORM_POS);
-    CollisionHeader_GetVirtual(D_8089EB70[MOVEBG_TYPE(thisx->params)], &colHeader);
+    CollisionHeader_GetVirtual(sColHeaders[MOVEBG_TYPE(thisx->params)], &colHeader);
     ((BgMizuMovebg*)thisx)->dyna.bgId = DynaPoly_SetBgActor(play, &play->colCtx.dyna, thisx, colHeader);
 
     type = MOVEBG_TYPE(thisx->params);
     switch (type) {
-        case 0:
-            temp = waterBoxes[2].ySurface + 15.0f;
-            if (temp < ((BgMizuMovebg*)thisx)->homeY - 700.0f) {
+        case MIZUMOVEBG_TYPE_FLOATING_PLATFORM_OUTSIDE_CENTER_PILLAR:
+            targetPosY = waterBoxes[2].ySurface + 15.0f;
+            if (targetPosY < ((BgMizuMovebg*)thisx)->homeY - 700.0f) {
                 thisx->world.pos.y = ((BgMizuMovebg*)thisx)->homeY - 700.0f;
             } else {
-                thisx->world.pos.y = temp;
+                thisx->world.pos.y = targetPosY;
             }
-            ((BgMizuMovebg*)thisx)->actionFunc = func_8089E318;
+            ((BgMizuMovebg*)thisx)->actionFunc = BgMizuMovebg_UpdateMain;
             break;
-        case 1:
-            temp = waterBoxes[2].ySurface + 15.0f;
-            if (temp < ((BgMizuMovebg*)thisx)->homeY - 710.0f) {
+        case MIZUMOVEBG_TYPE_FLOATING_PLATFORM_WEST:
```

Comment:
also not your change, but can you add a new line below every break in a switch statement?
We like to use a line break to separate cases, unless it is a fallthrough situation.

This may be relevant in other switches in the file, if there are any